### PR TITLE
I214 event names with long titles cause the chips to be cut off on mobile

### LINKED ISSUE #214: Event names with long titles cause the chips to be cut off on mobile
On the events page, a long event name will cause the chip showing "official" or "private" to be cut off

### PR DESCRIPTION
**Pull request by HuxleyBerry**
## Change Summary

**[Briefly summarise the changes that you made. Just high-level stuff]**
* Added some styles to the v-card-title so that the title will wrap

### Change Form

**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information

**[Is there anything in particular in the review that I should be aware of?]**

# Related Issue

- Resolve #214

### CODE REVIEW

File: client/src/components/EventCard.vue [hunk 1]
```diff
@@ -1,21 +1,17 @@
 <template>
   <v-card class="mx-3 elevation-0" :class="backgroundColour">
-    <v-card-title>
-      <span class="font-weight-bold">{{ event.name }}</span>
-      <v-spacer />
-      <v-icon
-        v-if="user?.teamAdmin && !event.isPublic"
-        icon="mdi-pencil"
-        @click="openModal"
```
Comment (ChantelleKerr) [inline client/src/components/EventCard.vue:9]: shouldnt we keep the v-if? We dont want any user editing public events and we dont want users that arent team admins to edit private events.

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
Small change

File: client/src/components/EventCard.vue (hunk 1 above)
Comment (HuxleyBerry) [reply client/src/components/EventCard.vue:9]: Oops, I had put the v-if in a v-col containing the icon, but when I got rid of the v-col I must have forgotten to put it back. Will fix.

**Review by ChantelleKerr — APPROVED** [review]
Nice

**Pull request merged**